iot_verifier/src/reward_share.rs

```diff
@@ -329,66 +340,68 @@ impl GatewayShares {
             reward_period.end - reward_period.start,
             dc_transfer_rewards_unused,
         );
-
         // work out the rewards per share for beacons, witnesses and dc transfer
         let beacon_rewards_per_share = rewards_per_share(total_beacon_rewards, total_beacon_shares);
         let witness_rewards_per_share =
             rewards_per_share(total_witness_rewards, total_witness_shares);
         let dc_transfer_rewards_per_share =
             rewards_per_share(total_dc_transfer_rewards_capped, total_dc_shares);
-        // compute the awards per hotspot
+
         tracing::info!(
             %total_dc_shares,
             %total_dc_transfer_rewards_used,
             %dc_transfer_rewards_unused,
             %dc_transfer_rewards_per_share,
             "data transfer rewards"
         );
+        self.beacon_rewards_per_share = beacon_rewards_per_share;
```

Comment:
Are these values defaulted before this function is called? If so, this function should be a constructor, not a &mut function. Could be a pretty dangerous footgun

Reply:
Addressed in sep PR via this commit: d1238dc8e5e255b4aa03f3c4a41fc6da62053b04